File: boj8/13241.cpp
// 최소공배수
// https://www.acmicpc.net/problem/13241

#include <stdio.h>

long long Gcd(long long x, long long y) {
  if (y == 0) return x;
  return Gcd(y, x % y);
}

long long Lcm(long long x, long long y) { return x * y / Gcd(x, y); }

int A, B;

int main() {
  scanf("%d%d", &A, &B);
  printf("%lld\n", Lcm(A, B));
  return 0;
}

File: past_archive/23boj_py/13241.py
# 최소공배수
# https://www.acmicpc.net/problem/13241

import sys


def GCD(a, b):
    if b == 0:
        return a

    return GCD(b, a % b)


a, b = map(int, sys.stdin.readline().split())

k = GCD(a, b)

sys.stdout.write(f'{a * b // k}\n')
